### ISSUE TITLE
Allow using directive body content as template

### ISSUE THREAD

**Issue by janbrohl** [sphinx-contrib/datatemplates#22]
Currently the room "inside" the directive is unused. 
Would it make sense to either make
```
.. datatemplate::
    :source: some-file.json
    
    {{ data.key }}
```
or
```
.. datatemplate::
    :template: great-template
    
    {"a":1,"b":2}
```
work?

**Comment by dhellmann** [comment]
I like that.

**Comment by janbrohl** [comment]
Actually, depening on which option is present, the body could be treaded as the missing part but that would be confusing, I guess.
Thinking over it again, putting the template string in the body seems more natural for me.

**Comment by janbrohl** [comment]
This could complement #34

**Comment by janbrohl** [comment]
Closed via  #36
